designdocs/capacity.md

```diff
@@ -23,89 +23,105 @@ Frequency explains how Capacity can be acquired through staking, refills, and us
 - [Block space allocation for Capacity transactions](#block-space)
 - [Implementation of spending Capacity to perform transactions](#capacity-transactions)
 
-  **Implementation of how to acquire through staking:** <a id='staking'></a>
+### **Implementation of how to acquire Capacity through staking:** <a id='staking'></a>
 
 This section is limited to the interfaces for staking and un-staking tokens.
 
 As a Registered Provider, you can receive Capacity by staking your tokens to the network or when others stake their tokens to the network.
 
-When staking tokens to the network, the network generates Capacity based on a Capacity-generating function that considers usage and other criteria. When you stake tokens, you will also provide a target Registered Provider to receive the Capacity generated. In exchange for staking Token to the network, you receive rewards. Rewards are deferred to a supplemental [staking design doc](https://github.com/frequency-chain/frequency/issues/40). You may increase your stake to network many times and target different Service Providers each time you stake. Note every time you stake to network your tokens are locked until you decide to unstake.
+When staking tokens to the network, the network generates Capacity based on a Capacity-generating function that considers usage and other criteria. When you stake tokens, you will also provide a target Registered Provider to receive the Capacity generated. In exchange for staking Token to the network, you receive rewards. For more information on rewards, please see the [Tokenomics docs](https://docs.frequency.xyz/Tokenomics/index.html). You may increase your stake to network many times and target different Service Providers each time you stake. Note every time you stake to network your tokens are frozen until you decide to unstake.
 
-Unstaking tokens allow you to schedule a number of tokens to be unlocked from your balance. There is no limit on the amount that you can schedule to be unlocked (up to the amount staked), but there is a limit on how many scheduled requests you can make. After scheduling tokens to be unlocked using **`unstake`**, you can withdraw those tokens after a thaw period has elapsed by using the **`withdraw_unstaked`** extrinsic. If the call is successful, all thawed tokens become unlocked and increase the ability to make more scheduled requests.
+Unstaking tokens allow you to schedule a number of tokens to be unfrozen from your balance. There is no limit on the amount that you can schedule to be unfrozen (up to the amount staked), but there is a limit on how many scheduled requests you can make. After scheduling tokens to be unfrozen using **`unstake`**, you can withdraw those tokens after a thaw period has elapsed by using the **`withdraw_unstaked`** extrinsic. If the call is successful, all thawed tokens become unfrozen and increase the ability to make more scheduled requests.
```

Comment:
Note: I tried to be consistent and use the verb `unfreeze` when referring to `unlock` and keeping `thaw` to reference the period of time required before `unfreezing`.

Reply:
Good catch since this was written before the Fungible trait change.